system_tests/batch_poster_test.go

```diff
@@ -144,22 +144,22 @@ func TestBatchPosterLargeTx(t *testing.T) {
 
 	conf := arbnode.ConfigDefaultL1Test()
 	conf.Sequencer.MaxTxDataSize = 110000
-	l2info, nodeA, l2clientA, l1info, _, _, l1stack := createTestNodeOnL1WithConfig(t, ctx, true, conf, nil, nil)
-	defer requireClose(t, l1stack)
-	defer nodeA.StopAndWait()
+	testNodeA := NewNodeBuilder(ctx).SetNodeConfig(conf).SetIsSequencer(true).CreateTestNodeOnL1AndL2(t)
+	defer requireClose(t, testNodeA.L1Stack)
+	defer testNodeA.L2Node.StopAndWait()
 
-	l2clientB, nodeB := Create2ndNode(t, ctx, nodeA, l1stack, l1info, &l2info.ArbInitData, nil)
+	l2clientB, nodeB := Create2ndNode(t, ctx, testNodeA.L2Node, testNodeA.L1Stack, testNodeA.L1Info, &testNodeA.L2Info.ArbInitData, nil)
```

Comment:
I think that since we use the same builder for L1 and L2 - it should also be possible to use it for building / holding / etc multiple L2 nodes. So there should be a way to build and retrieve a 2nd node from the builder itself.